packages/block-editor/src/components/font-appearance-control/index.js

```diff
@@ -110,7 +110,7 @@ export default function FontAppearanceControl( props ) {
 							? weightName
 							: sprintf(
 									/* translators: 1: Font weight name. 2: Font style name. */
-									__( '%1$s %2$s' ),
+									_x( '%1$s %2$s', 'font weight and style' ),
```

Comment:
We use this `%1$s %2$s` in multiple places. Without context it's impossible for translators to know whether their order should be reversed or not - or worse, it may be reversed everywhere instead of just a single place.